### PR TITLE
EOL

### PR DESCRIPTION
**Pull request by oOKexOo**
**When merged this pull request will:**
- Announce EOL :tada: 
- Update change logs and credits

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,11 @@
     </a>
 </p>
 
+## IMPORTANT ANNOUNCEMENT
```
Comment (CreepPork) [inline README.md]: Could be Depreciation Notice

Comment (bovine3dom) [reply README.md]: FWIW, it's spelt "deprecation", depreciation is a different word :)

Comment (oOKexOo) [reply README.md]: Sounds good when spelled correctly :wink:

Comment (CreepPork) [reply README.md]: Welp, that's what swipe type does :D

**Pull request merged**